src/tctrack/track/track.py

```diff
@@ -367,6 +380,53 @@ def _get_filter_trajectories_inputs(self) -> list[str]:
 
         return inputs
 
+    def _export_or_read_inputs(self, command_name: str, inputs: list[str]) -> str:
+        """Reads/exports the command line inputs to a file and returns the command list.
+
+        If :attr:`~TRACKParameters.read_inputs` is ``True`` but the file doesn't exist a
+        warning will be thrown and it will continue with the generated inputs.
+
+        Parameters
+        ----------
+        command_name : str
+            The name of the TRACK step. Used in the file name:
+            ``<inputs_directory>/<command_name>.in``.
+        inputs : list[str]
+            The list of input parameters to pass to the TRACK program.
+
+        Returns
+        -------
+        str
+            A string containing the input parameters separated by newline characters.
+        """
+        input_commands = "\n".join(inputs) + "\n"
```

Comment:
This function is named `_export_or_read_inputs`, but it could do neither of these and has an additional function of creating the inputs list from user-supplied values.

Perhaps rename to "create_inputs_string" or similar, or separate out the read/write from the generation from user files.